account-kit/smart-contracts/src/ma-v2/modules/single-signer-validation/signer.ts

```diff
@@ -112,8 +112,11 @@ export const singleSignerMessageSigner = (
     ): Promise<Hex> => {
       // the accounts domain already gives replay protection across accounts for deferred actions, so we don't need to apply another wrapping
       const isDeferredAction =
-        typedDataDefinition?.primaryType === "DeferredAction" &&
-        typedDataDefinition?.domain?.verifyingContract === accountAddress;
+        typedDataDefinition.primaryType === "DeferredAction" &&
+        typedDataDefinition.domain != null &&
+        // @ts-expect-error the domain type I think changed in viem, so this is not working correctly
```

Comment:
Odd... can you leave a todo here? I think we ran into this issue in a few other places, too.

Reply:
yea can do